src/cargo/core/compiler/build_config.rs

```diff
@@ -41,7 +41,7 @@ pub struct BuildConfig {
     /// Output a build plan to stdout instead of actually compiling.
     pub build_plan: bool,
     /// An optional override of the rustc path for primary units only
-    pub primary_unit_rustc: Option<ProcessBuilder>,
+    pub rustc_wrapper: Option<ProcessBuilder>,
```

Comment:
I think I would leave the name here the same, to make it clear it would only be used for primary packages.

Reply:
Okay, but I need to fix it to apply only to primary crates, other changes I made to get it working initially made it a straight override